lib/commands/binlog_dump.js

```diff
@@ -60,6 +98,147 @@ function QueryEvent(packet) {
   this.name = 'QueryEvent';
 }
 
+function readPackedInteger (packet) {
```

Comment:
is this the same as `packet.readLengthCodedNumber` ?

Reply:
yes, I don't know that method exists, so I write my own one. I will try to use that one.